source/scenario/mission3task1.vwf.yaml

```diff
@@ -163,6 +163,15 @@ children:
               - enabled
               - true
 
+        setCamera_3_1:
+          triggerCondition:
+          - onScenarioStart:
+          actions:
+          - setCinematicCameraView:
+            - [ 16, 160, -15 ]
+          - setThirdPersonStartPose:
+            - [ 16, 160, -15 ]
```

Comment:
You should just be able to put this up in the start state, rather than adding an onScenarioStart trigger for it.  It's a nitpick, but a bit more efficient (and clearer).